js/scenes/ProjectConfirm/styles.js

```diff
@@ -1,3 +1,19 @@
 import { StyleSheet } from "react-native";
-
-export const styles = StyleSheet.create({});
+import { colors, typography } from "../../config/styles.js";
+const { black, mediumGrey, nearBlack, green, red, white } = colors;
```

Comment:
remove the colors that you are not using. How about calling it "tintBlack" instead of "nearBlack"?